frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -111,7 +111,7 @@ const Notifications: React.FC<NotificationsProp> = ({ data }) => {
                 <MenuList
                   autoFocusItem={open}
                   id="notification-options"
-                  onKeyDown={handleListKeyDown}
+                  onKeyDown={() => handleListKeyDown}
```

Comment:
This one is a bit confusing. Should this be `onKeyDown={() => handleListKeyDown()}` or remain the same?

Reply:
Yeah that's a good catch, was focused on the lint issue and missed that. Fixed!

Reply:
cool I think it's in a few spots. I just commented on this one

Reply:
Yup, I did a find/replace on all of them to make sure I caught them